requirements/dev.in

```diff
@@ -10,6 +10,10 @@ PyPOM
 # for git commit hooks
 pre-commit
 
+# for test driven development
+pytest-testmon
+pytest-watch
```

Comment:
Test runners optimized for test driven development

Reply:
@ionutgoldan  Please add the `ptw --runner “pytest --testmon”` command to the `tests/README.md` in the correct spot